src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -81,6 +85,32 @@ public long claimCount(UUID playerUUID) {
         return getClaimsOf(playerUUID).size();
     }
 
+    @StringProvider(
+            text = "Claim Type",
+            description = "Basic + Town + SUBDIVISION + Admin",
+            iconName = "map-marker",
+            iconColor = Color.BLUE,
+            showInPlayerTable = true
+    )
+    public String claimType(UUID playerUUID) {
+        if (getClaimsOf(playerUUID) != null) {
+            getClaimsOf(playerUUID)
+                    .forEach(this::countType);
+        }
+        return b + "+" + t + "+" + s + "+" + a ;
+    }
+
+    int b=0, t=0, s=0, a=0;
```

Comment:
Please name these variables and move them inside a function - currently they are declared as class variables.
Good names tell what the variable is about.